src/content/language/functions.md

```diff
@@ -49,9 +49,13 @@ The <code>=> <em>expr</em></code> syntax is a shorthand for
 is sometimes referred to as _arrow_ syntax.
 
 :::note
-Only an *expression*—not a *statement*—can appear between the arrow (=\>) and
-the semicolon (;). For example, you can't put an [if statement][]
-there, but you can use a [conditional expression][].
+Only _expressions_ can appear between the arrow (=\>) and the semicolon (;).
+Expressions return values.
+This means that you can't write code in a place that expects a value.
```

Comment:
I wouldn't say "code".  Expressions *are* code.  Maybe keep the reference to statements? e.g. "...you can't put a statement (for example an [if statement][]) after the arrow."